packages/material-ui/src/Checkbox/Checkbox.js

```diff
@@ -138,7 +138,7 @@ Checkbox.propTypes = {
   /**
    * The value of the component.
```

Comment:
Could we add some sort of hint that this is weakly typed i.e. someone (we? react? DOM api?) is casting the value to a string?

Reply:
Sure, I believe it's the DOM API. This change also changes value to match the TypeScript types.